### PR TITLE
dev/user-interface#51 Remove top buttons on admin forms

### PR DESCRIPTION
**Pull request by mlutfy**
Overview
----------------------------------------

https://lab.civicrm.org/dev/user-interface/-/issues/51

Before
----------------------------------------

image

image

image

After
----------------------------------------

image

image

Technical Details
----------------------------------------

Many forms open in popups, so the top buttons are often not visible anyway.

Some files have been deleted because I could not find where they are used:

- templates/CRM/Admin/Form/Setting/Event.tpl
- templates/CRM/Contribute/Form/PaymentInstrument.tpl
- templates/CRM/Contribute/Form/Preview.tpl

I found some dead forms, but did not remove them for now, because some seem to be in the default navigation menu:

- /civicrm/admin/report/register = CRM_Report_Form_Register (tpl and PHP) - accessible from /civicrm/admin/report/options/report_template?reset=1
- /civicrm/admin/paymentProcessorType = CRM_Admin_Page_PaymentProcessorType (tpl and PHP)

Comments
-------------

I might not remove the top buttons on a few pages that I know can be very long, such as:

- List of Message Templates
- List of Profiles

I was not able to test Survey > Release Respondents. I couldn't figure out how to make it work (it would bounce me to the Advanced Search).

### CODE REVIEW

**Comment by magnolia61** [comment]
👍

**Comment by mlutfy** [comment]
Ready for review :tada:

There are a few "top" buttons left, either because it's EventInfo, or I wasn't sure how to reach that part of the code, or I was done procrastinating.

I did test most of the changes. I may have cut corners in a few places, where I was fairly confident it was fine (like in the Import forms, I tested only Contact Import).

I also updated sometimes the "status" markup, because it's something I'd like to also address eventually (https://lab.civicrm.org/dev/user-interface/-/issues/52).

File: templates/CRM/Admin/Form/Setting/Event.tpl [hunk 1]
```diff
@@ -1,26 +0,0 @@
...
-<div class="crm-submit-buttons">{include file="CRM/common/formButtons.tpl" location="top"}</div>
-      <table class="form-layout-compressed">
-        <tr class="crm-mail-form-block-enable_cart">
-            <td class="label">{$form.enable_cart.label}</td><td>{$form.enable_cart.html}<br />
-            <span class="description">{ts}Check to enable the Event Cart checkout workflow.{/ts}</span></td>
-        </tr>
-      </table>
-<div class="crm-submit-buttons">{include file="CRM/common/formButtons.tpl" location="bottom"}</div>
-<div class="spacer"></div>
-</div>
```
Comment (colemanw) [inline templates/CRM/Admin/Form/Setting/Event.tpl:26]: @mlutfy why was this file deleted?

File: templates/CRM/Contribute/Form/PaymentInstrument.tpl
Comment (colemanw) [inline templates/CRM/Contribute/Form/PaymentInstrument.tpl:1]: @mlutfy why was this file deleted?

File: templates/CRM/Contribute/Form/Preview.tpl
Comment (colemanw) [inline templates/CRM/Contribute/Form/Preview.tpl:1]: And this one?

**Comment by mlutfy** [comment]
@colemanw I could not find any references to the deleted files.

- `templates/Contribution/Form/Preview.tpl` - has fields for things like cvv2, it looks very legacy
- `templates/CRM/Admin/Form/Setting/Event.tpl` - couldn't find any reference either, there is another tpl for CiviEvent settings, which includes the Event Cart setting.

I have to admit I have doubts about `templates/CRM/Contribute/Form/PaymentInstrument.tpl`, but it's an oddly named template. There is nothing under Manage Contribution Page, nor in places such as `/civicrm/admin/options/payment_instrument?action=add&reset=1`.

(p.s. this was mentioned in the issue summary)

**Comment by colemanw** [comment]
Ok I just double checked and I agree they are unused.
